newsplease/crawler/commoncrawl_extractor.py

```diff
@@ -226,10 +200,16 @@ def __download(self, url):
                 pass
 
             # download
-            self.__logger.info('downloading %s (local: %s)', url, local_filepath)
-            urllib.request.urlretrieve(url, local_filepath, reporthook=self.__on_download_progress_update)
-            self.__logger.info('download completed, local file: %s', local_filepath)
-            return local_filepath
+            if self.__s3_client:
+                with open(local_filepath, 'wb') as file_obj:
+                    self.__s3_client.download_fileobj(self.__cc_bucket, path, file_obj)
+                    return local_file_path
```

Comment:
I believe this is a typo and should be `local_filepath` (this fixed things for me when trying to run this fix.)

Reply:
Yes, of course!

Reply:
Updated the PR. Thanks, @eeegnu!